docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -287,6 +287,72 @@ Time zones may either be specified as an ISO 8601 UTC offset (e.g. `+01:00` or
 `-08:00`)  or as a timezone id, an identifier used in the TZ database like
 `America/Los_Angeles`.
 
+*Offset*
+
+include::datehistogram-aggregation.asciidoc[tag=offset-explanation]
```

Comment:
I figured because the option is the same I should just include the docs from the normal date histogram.